dbt/include/bigquery/macros/relations/materialized_view/alter.sql

```diff
@@ -0,0 +1,16 @@
+{% macro bigquery__get_alter_materialized_view_as_sql(
+    relation,
+    configuration_changes,
+    sql,
+    existing_relation,
+    intermediate_relation
+) %}
+    get_replace_materialized_view_as_sql(
```

Comment:
As noted above, replace this with `get_replace_sql`. This macro won't need intermediate_relation or backup_relation. What's temp_relation?